nms/app/packages/magmalte/app/components/context/RefreshContext.js

```diff
@@ -95,6 +95,12 @@ export function useRefreshingContext(props: Props) {
     return ctx.setState(null, null, newState);
   }
 
+  // Avoid using state as a dependency of useEffect
+  const stateRef = useRef(null);
+  useEffect(() => {
+    stateRef.current = state;
+  });
```

Comment:
```suggestion
  useEffect(() => {
    stateRef.current = state;
  }, [state]);
```

Should this only trigger on `state` updating?

Reply:
Yes you're right!

Reply:
Yes!